requirements.moreh.txt

```diff
@@ -0,0 +1,25 @@
+transformers==4.29.2
+accelerate==0.21.0
+datasets==2.14.3
```

Comment:
In the last discussion in Slack, we change dataset to version 2.14.6 to avoid an errors

Reply:
Ah okay. I will change it.